salt/modules/zabbix.py

```diff
@@ -1489,6 +1489,290 @@ def hostinterface_update(interfaceid, **connection_args):
         return ret
 
 
+def usermacro_get(macro=None, hostids=None, templateids=None, hostmacroids=None,
```

Comment:
Although I agree with you, this nomenclature follows the naming convention from the Zabbix API. See here for some examples:

https://www.zabbix.com/documentation/3.0/manual/api/reference/usermacro/object
https://www.zabbix.com/documentation/3.0/manual/api/reference/usermacro/get

Even though it looks ugly, for people with background knowledge of the Zabbix API it is immediately recognisable.